graph-service/enricher.py

```
@@ -0,0 +1,875 @@
+        if isinstance(output, SearchResultSelection):
+            return output.selected_urls
+        
+        # Fallback: return first 3 results
+        return [r["url"] for r in search_results[:3]]
+        
+    except Exception as e:
+        logger.error(f"Error selecting results: {e}")
+        # Fallback: return first 3 results
+        return [r["url"] for r in search_results[:3]]
+
+async def enrich_entity_page(entity_id: str, entity_name: str, entity_type: str, page_url: str, browser: Browser, entity_metadata: Optional[Dict[str, Any]] = None, entity_relationships: Optional[List[Dict[str, str]]] = None) -> Optional[EnrichmentResult]:
```

Ruff (0.14.7)

Unused function argument: `entity_id`

(ARG001)